debian/changelog

```diff
@@ -1,8 +1,66 @@
-ubuntu-advantage-tools (1:1+devel) UNRELEASED; urgency=medium
+ubuntu-advantage-tools (32) noble; urgency=medium
```

Comment:
This actually won't be released until Optical Orangutan is the new devel release, but since we don't know the real name, I'm not sure what to put here for now.

Reply:
@athos-ribeiro any suggestion here ?

Reply:
how about UNRELEASED for now since this is not going to be uploaded anywhere?

Reply:
OK. After talking to Lucas offline, I realize that the reasoning behid not using noble here is the concern that the 24.10 (oo) development cycle could open during the SRU process for this new release.

IMHO, let's not worry about it for now. You can keep `noble` here (we do need to push this to noble anyway).

If 24.10 opens for development __after__ this is accepted in -proposed, i suppose it will be picked up in OO. If it otherwise the oo development cycle opens __before__ this is accepted in proposed by the SRU team, then we can circle back and re-adjust (i.e., upload 32 to oo and a backport to noble).

Cc @panlinux

Reply:
OO will open up with whatever is in noble at that time i.e., binary copies. So yes, we will have u-a-t v32 in both noble and OO, and that is fine. +1 for the `noble` name.